packages/wonder-blocks-dropdown/src/components/__tests__/combobox.test.tsx

```diff
@@ -296,6 +305,109 @@ describe("Combobox", () => {
         expect(screen.getByRole("combobox")).not.toHaveFocus();
     });
 
+    it("should clear the value when the user presses the clear button (x)", async () => {
+        // Arrange
+        const userEvent = doRender(
+            <Combobox selectionType="single" value="option2">
+                <OptionItem label="option 1" value="option1" />
+                <OptionItem label="option 2" value="option2" />
+                <OptionItem label="option 3" value="option3" />
+            </Combobox>,
+        );
+
+        // Act
+        await userEvent.click(
+            screen.getByRole("button", {name: /clear selection/i}),
+        );
+
+        // Assert
+        expect(screen.getByRole("combobox")).toHaveValue("");
+    });
+
+    describe("error", () => {
+        it("should use aria-invalid=false by default", () => {
+            // Arrange
+
+            // Act
+            doRender(
+                <Combobox selectionType="single" value="">
+                    <OptionItem label="option 1" value="option1" />
+                    <OptionItem label="option 2" value="option2" />
+                    <OptionItem label="option 3" value="option3" />
+                </Combobox>,
+            );
+
+            // Assert
+            expect(screen.getByRole("combobox")).toHaveAttribute(
+                "aria-invalid",
+                "false",
+            );
+        });
+
+        it("should display the errror message when the value is invalid", async () => {
```

Comment:
question: Should the test be something like "should use aria-invalid=true if error is true" or something since it is checking for aria-invalid rather than an error message?